libraries/standard/json/src/json.c

```diff
@@ -394,7 +412,7 @@ static bool_ skipEscape( const char * buf,
             default:
 
                 /* a control character: (NUL,SPACE) */
-                if( ( ( uint8_t ) c ) < 0x20U )
+                if( ( c > '\0' ) && ( c < ' ' ) )
```

Comment:
You have macros for other character sets. One here would make things clearer.

Reply:
Added